src/Microsoft.ApplicationInsights.AspNetCore/project.json

```diff
@@ -101,8 +101,7 @@
         "netstandard1.6": {
             "dependencies": {
                 "NETStandard.Library": "1.6.1",
-                "System.Net.NameResolution": "4.0.0",
-                "System.Diagnostics.StackTrace": "4.3.0"
+                "System.Net.NameResolution": "4.3.0"
```

Comment:
We will bring `StackTrace` by this package: https://github.com/Microsoft/ApplicationInsights-dotnet/blob/fcee8b33191789c1ffba4eebcde44856591b4a91/src/Core/NuGet/Package.nuspec#L35-L37 Is it going to be a problem?